ksql-functional-tests/src/main/java/io/confluent/ksql/test/tools/TestExecutor.java

```diff
@@ -80,7 +80,6 @@
       .put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest")
       .put(StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG, 0)
       .put(KsqlConfig.KSQL_SERVICE_ID_CONFIG, "some.ksql.service.id")
-      .put(StreamsConfig.TOPOLOGY_OPTIMIZATION, "all")
```

Comment:
This was nasty: test case was explicitly turning on optimizations. Meaning the bug introduced when the optimization was turned of in `KsqlConfig` was not found.